curvefs/src/client/inode_cache_manager.h

```diff
@@ -285,6 +290,10 @@ class InodeCacheManagerImpl : public InodeCacheManager,
     std::map<uint64_t, std::shared_ptr<InodeWrapper>> dirtyMap_;
     curve::common::Mutex dirtyMapMutex_;
 
+    // record opened inode
+    std::multiset<uint64_t> openedInodes_;
```

Comment:
Why use multiset here, and what's the behavior if two processes open the same file in curvefs? Does curvefs receive two open calls of the same file?

Reply:
If two processes open the same file in curvefs, the curvefs will receive two open calls on the same inode.